lua/alex/lazy/lsp.lua

```
@@ -109,8 +69,30 @@ return {
   },
   config = function(_, opts)
     local lspconfig = require('lspconfig')
+
+    -- Defines the on_attach function
+    local on_attach = function(client, bufnr)
```

Luacheck (1.2.0)

[warning] unused argument 'client'

(W212)